src/types/user.types.ts

```diff
@@ -3,7 +3,10 @@ import { Keywords } from './keywords.types';
 
 export enum RoleName {
   AppAdmin = 'App-admin',
-  Curator = 'Curator',
+  Curator = 'Curator', // TODO: remove
```

Comment:
Tror det er best å vente til backend har migrert roller før jeg fjerner dette. Samme for den lenger ned her